PhysicsTools/NanoAOD/python/nano_cff.py

```diff
@@ -284,3 +284,4 @@ def nanoL1TrigObjCustomizeFull(process):
     process.nanoTableTaskCommon.add(process.l1TablesTask)
     return process
 
+from PhysicsTools.NanoAOD.custom_muon_cff import *
```

Comment:
please remove this line from nano_cff, and include it in autoNano (see the relevant comment)

Reply:
Done